scripts/analyze.py

```diff
@@ -13,10 +14,28 @@ def get_project_fuzzer_stats(project: str) -> pd.DataFrame:
     proj = Project(project, fuzzout, dumpout)
     return proj.get_project_stats()
 
+def get_proj_name(line: str) -> Optional[str]:
+    #pattern = '^[0-9]+\. \[?([A-Za-z0-9\-\_]+)\]?\([A-Za-z\.\_\-/ +]+\),([A-Za-z+]+)'
+    # Group 1 is project name, group 2 is project language
+    pattern = '^[0-9]+\. \[?([^\]]+)\]?\(.+\),([A-Za-z+]+)'
+    
+    line = line.strip()  # remove space at front and end
+    match = re.match(pattern, line)
+
+    if not match:
+        return None
+    
+    proj_name = match.group(1)
+    proj_language = match.group(2)
+    return proj_name
```

Comment:
actually, this function might `return proj_name, proj_language`, so the return type is `Optional[Tuple[str, str]]`. in this way we can filter on the language later